OSDOCS-20733: CQA fixes for ROSA HCP Tutorials 1-6 (#115626)
openshift/openshift-docs: modules/subscription-enablement.adoc

:_mod-docs-content-type: PROCEDURE
[id="subscription-enablement_{context}"]
= Enable the subscription and set up your AWS account

[role="_abstract"]
Activate {product-title} on the AWS console page to enable billing and metering for the service.

.Prerequisites

> * You're logged in to the Red{nbsp}Hat account that you want to associate with the AWS account that will activate the {product-title} product subscription.

Reviewer: avoid contractions

```suggestion
* Log into the Red{nbsp}Hat account that you want to associate with the AWS account that activates the {product-title} product subscription.
```

Author: I believe RH is turning towards using contractions but I'd double-check that first. Will keep a full form for now.
